tdesign/src/Layout/Header/Header.tsx

```diff
@@ -133,23 +134,22 @@ const Header = ({ children }: HeaderProps) => {
               ":checked": {
                 ...iconCheckedStyle,
                 "~ div.header--center": {
-                  gridRow: "2",
-                  gridColumnStart: "1",
-                  gridColumnEnd: "none",
-                  width: "100%",
-                  paddingLeft: "22.5px",
-                  paddingRight: "22.5px",
-                  "> .search-input-container": {
+                  [theme.breakpoints.down("sm")]: {
+                    gridRow: "2",
+                    gridColumnStart: "1",
+                    gridColumnEnd: "none",
                     width: "100%",
+                    paddingLeft: "22.5px",
+                    paddingRight: "22.5px",
+                  },
+                  "~ div.header--right": {
```

Comment:
Is it expected that this style is now nested inside `~ div.header--center`? I see none of the values changed, and I suspect this may be a mistake that it is now nested inside of another ruleset

Reply:
Hmm- A mistake when I added the `breakpoint`  that seemed to render fine 😅

 `~ div.header--right` should be a sibling to `~ div.header--center`